info.limpet.stackedcharts.ui/src/info/limpet/stackedcharts/ui/editor/parts/ChartEditPart.java

```diff
@@ -197,17 +198,40 @@ public void notifyChanged(Notification notification)
     {
       StackedchartsPackage pckg = StackedchartsPackage.eINSTANCE;
       Object feature = notification.getFeature();
```

Comment:
@IanMayo I'm OK with the suggested implementation. What we can do in addition is initialize the visualUpdates and childrenUpdates lists in the ChartAdapter constructor. No need to initialize it each time in notifyChanged.